infra/addSecrets.bicep

```diff
@@ -0,0 +1,19 @@
+param KeyVaultName string = 'Kv-SSW-v3'
```

Comment:
name should be parameterized and follow naming conventions consistent with the rest of the project - see rules at <https://www.ssw.com.au/rules/azure-naming-resources>

since its a kv - I suggest adding entropy to the name in the same way the app service has entropy